android/app/src/main/java/org/fossasia/openevent/fragments/SpeakersListFragment.java

```diff
@@ -153,15 +146,14 @@ public void onDestroyView() {
         Utils.unregisterIfUrlValid(this);
 
         // Remove listeners to fix memory leak
-        realmResults.removeAllChangeListeners();
         if(swipeRefreshLayout != null) swipeRefreshLayout.setOnRefreshListener(null);
         if(searchView != null) searchView.setOnQueryTextListener(null);
     }
 
     @Override
     public void onSaveInstanceState(Bundle bundle) {
```

Comment:
No need for this anymore, ViewModel will outlive the activity lifecycle so you'll always get the same data you fetched initially.